examples/plugin/com_osvr_example_Tracker.cpp

```diff
@@ -0,0 +1,122 @@
+/** @date 2015
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2015 Sensics Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Internal Includes
+#include <osvr/PluginKit/PluginKit.h>
+#include <osvr/PluginKit/TrackerInterfaceC.h>
+
+// Generated JSON header file
+#include "com_osvr_example_Tracker_json.h"
```

Comment:
Recommending all new plugins be named `org_osvr_whatever` - and if this is intended as a dummy plugin, not just an example (since there are examples that put out tracker data, aren't there?), maybe namespace that name differently.